client/src/components/Login/Login.tsx

```diff
@@ -5,7 +5,7 @@ import {
 
 function Login() {
   const connectSpotify = () => {
-    window.location.href = 'http://localhost:9000/auth/login';
+    window.location.href = 'http://localhost:9000/auth/login'; // is there a way to thunk this?
```

Comment:
I guess you could try. Is this moreso to consolidate code/make it DRYer upon eventual reuse, or is it so you can trigger some behavior after the redirect happens? Not sure really what will be gained. Technically changing the `.href` property *is* basically asynchronous, so maybe it might work? Dunno.